app/src/page/firstRun/view.js

```diff
@@ -188,10 +236,26 @@ class FirstRunScreen extends React.PureComponent {
           {emailNewPending &&
             <ActivityIndicator size="small" color={Colors.White} style={firstRunStyle.pageWaiting} />}
 
-          {!emailNewPending &&
-          <TouchableOpacity style={firstRunStyle.button} onPress={this.handleContinuePressed}>
-            <Text style={firstRunStyle.buttonText}>{this.state.showSkip ? 'Skip': 'Continue'}</Text>
-          </TouchableOpacity>}
+          <View style={firstRunStyle.buttonRow}>
+            {([Constants.FIRST_RUN_PAGE_WELCOME, Constants.FIRST_RUN_PAGE_WALLET].indexOf(this.state.currentPage) > -1) && <View />}
```

Comment:
This would be much more legible if you use variables for some of the rendered output.  Up to you